.github /ISSUE_TEMPLATE/otel_q_and_a.md

```diff
@@ -0,0 +1,32 @@
+---
+name: OTel Q&A
+about: Issue tracking a new episode of OTel Q&A
+title: 'OTel Q&A: <name>'
+labels: OTel Q&A
+assignees: ''
+
+---
+<!-- Please remember to change the title of this issue by replacing <name> with the actual name for the session. This will be the title used in event invites -->
+
+### Date and time for the session
+<!-- Add here the proposed date for the event after it's been agreed -->
+
+TBD
+
+### Description used for events
+<!-- Add here a description that will be used in events generated for the event -->
+
+### Tasks
+The following list contains sub-tasks that need to be completed as part of organising, recording, and publicising the episode. Please replace items with individual links to issues where appropriate.
+
+- [ ] Confirm speakers
+- [ ] Confirm date and time
+- [ ] Create Q&A doc
```

Comment:
Suggest replacing with: 

- [ ] Make a copy of the [Q&A interview template](https://docs.google.com/document/d/1vi6NpSj3MvKTuNwFpE8toywHP0sWiH4bsE3AOWvgSwo/edit)
- [ ] Fill in the template fields highlighted in blue